pkg/gcp/machine_controller_test.go

```diff
@@ -70,6 +71,10 @@ const (
 	FailAtSpecValidation string = "machine codes error: code = [Internal] message = [Create machine \"dummy-machine\" failed on decodeProviderSpecAndSecret: machine codes error: code = [Internal] message = [Error while validating ProviderSpec [spec.zone: Required value: zone is required]]]"
 	// FailAtNonExistingMachine because existing machine is not found
 	FailAtNonExistingMachine string = "rpc error: code = NotFound desc = Machine with the name \"non-existent-dummy-machine\" not found"
+	// FailAtSpecValidationNoKmsServiceAccount if kmsKeyName specified but kmsKeyServiceAccount missing
+	// FailAtSpecValidationNoKmsServiceAccount if kmsKeyName specified but kmsKeyServiceAccount missing
+	FailAtSpecValidationNoKmsServiceAccount string = "machine codes error: code = [Internal] message = [Create machine \"dummy-machine\" failed on decodeProviderSpecAndSecret: machine codes error: code = [Internal] message = [Error while validating ProviderSpec [spec.disks[0].kmsKeyServiceAccount: Required value: kmsKeyServiceAccount is required to be specified along with kmsKeyName]]]"
+	FailAtSpecValidationNoKmsKeyName        string = "machine codes error: code = [Internal] message = [Create machine \"dummy-machine\" failed on decodeProviderSpecAndSecret: machine codes error: code = [Internal] message = [Error while validating ProviderSpec [spec.disks[0].kmsKeyServiceAccount: Required value: kmsKeyName is required to be specified along with kmsKeyServiceAccount]]]"
```

Comment:
Minor nit pick. 
```suggestion
	// FailAtSpecValidationNoKmsServiceAccount if kmsKeyName is specified but kmsKeyServiceAccount is missing
	FailAtSpecValidationNoKmsServiceAccount string = "machine codes error: code = [Internal] message = [Create machine \"dummy-machine\" failed on decodeProviderSpecAndSecret: machine codes error: code = [Internal] message = [Error while validating ProviderSpec [spec.disks[0].kmsKeyServiceAccount: Required value: kmsKeyServiceAccount is required to be specified along with kmsKeyName]]]"
	// FailAtSpecValidationNoKmsName if kmsKeyServiceAccount is specified but kmsKeyName is missing
	FailAtSpecValidationNoKmsKeyName        string = "machine codes error: code = [Internal] message = [Create machine \"dummy-machine\" failed on decodeProviderSpecAndSecret: machine codes error: code = [Internal] message = [Error while validating ProviderSpec [spec.disks[0].kmsKeyServiceAccount: Required value: kmsKeyName is required to be specified along with kmsKeyServiceAccount]]]"
```

Reply:
Just for completeness sake that it caught your eye as well, as I saw the commit mentions `Error message grammar`. 
The Lines 74/75 in the original were duplicated without any change to text and were also placed wrongly.